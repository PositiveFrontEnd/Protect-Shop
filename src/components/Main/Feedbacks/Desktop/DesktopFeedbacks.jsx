import React, { useEffect, useState } from "react";
import "./DesktopFeedbacks.scss";
import Button from "../../../Button/Button";
import { useNavigate } from "react-router";
import DesktopFeedbackCard from "./desktopCard.jsx";
const DesktopFeedbacks = ({ comments }) => {
  const navigate = useNavigate();
  const [randomComments, setRandomComments] = useState([]);
  useEffect(() => {
    const shuffleArray = (array) => {
      const newArray = [...array];
      let currentIndex = newArray.length,
        temporaryValue,
        randomIndex;
      while (0 !== currentIndex) {
        randomIndex = Math.floor(Math.random() * currentIndex);
        currentIndex -= 1;

        temporaryValue = newArray[currentIndex];
        newArray[currentIndex] = newArray[randomIndex];
        newArray[randomIndex] = temporaryValue;
      }
      return newArray;
    };

    const shuffledComments = shuffleArray(comments);

    const selectedComments = shuffledComments.slice(0, 3);

    setRandomComments(selectedComments);
  }, [comments]);
  return (
    <div className="feedbacks__wrapper">
      <div className="container feedbacks">
        <h2 className="feedbacks__title">feedbacks</h2>
        <div className="feedbacks__content">
          {randomComments.map((item, index) => (
            <DesktopFeedbackCard
              key={index}
              name={item.customer?.firstName || 'Unknown User'}
              avatar={item.customer?.avatarUrl}
              likes={item.likes}
              comment={item.advantages}
              background={item.customer?.background || ''}
            />
          ))}
        </div>
        <div className="feedbacks__button">
          <Button click={() => navigate("/feedbacks")} white>
            see more
          </Button>
        </div>
      </div>
    </div>
  );
};

export default DesktopFeedbacks;
