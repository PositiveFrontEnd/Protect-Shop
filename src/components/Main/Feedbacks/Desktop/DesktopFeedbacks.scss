.feedbacks__wrapper {
  background-color: var(--white);
  padding-block: 60px;
}

.feedbacks {
  display: flex;
  flex-direction: column;

  .feedbacks__title {
    font-size: 24px;
    font-family: "Noto Sans", sans-serif;
    text-align: center;
    text-transform: uppercase;
  }

  .feedbacks__content {
    display: flex;
    gap: 12px;
    justify-content: center;
    padding-block: 60px 40px;

    .feedbacks__card {
      display: flex;
      flex-direction: column;
      align-items: center;
      padding: 20px 40px;
      background-color: var(--feedback);
      border-radius: 2px;

      .feedbacks__stars {
        display: flex;
        gap: 12px;
        margin-top: 32px;
      }

      .feedbacks__card__name {
        margin-top: 36px;
        font-size: 18px;
        text-transform: uppercase;
      }

      .feedbacks__card__description {
        margin-top: 20px;
        font-size: 14px;
      }
    }
  }

  .feedbacks__button {
    align-self: center;
    width: 100%;
    max-width: 280px;
  }
}
