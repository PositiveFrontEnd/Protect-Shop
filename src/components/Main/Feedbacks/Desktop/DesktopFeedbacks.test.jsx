import React from "react";
import { render } from "@testing-library/react";
import { Provider } from 'react-redux';
import DesktopFeedbacks from "./DesktopFeedbacks";
import { BrowserRouter } from "react-router-dom";
import configureStore from "redux-mock-store";

describe("Тестування компонента  DesktopFeedbacks", () => {
    test("Знімок", () => {
        const mockStore = configureStore();
        const initialState = {

        };

        const store = mockStore(initialState);
        const desktopFeedbacks = render(
            <Provider store={store}>
                <BrowserRouter>
                    <DesktopFeedbacks />
                </BrowserRouter>
            </Provider>
        );

        expect(desktopFeedbacks).toMatchSnapshot();
    });
});
