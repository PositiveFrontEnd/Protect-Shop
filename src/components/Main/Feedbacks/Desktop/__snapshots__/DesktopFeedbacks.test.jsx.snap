// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Тестування компонента  DesktopFeedbacks > Знімок 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="feedbacks__wrapper"
      >
        <div
          class="container feedbacks"
        >
          <h2
            class="feedbacks__title"
          >
            feedbacks
          </h2>
          <div
            class="feedbacks__content"
          >
            <div
              class="feedbacks__card"
            >
              <div
                class="feedbacks__image"
              >
                <img
                  alt=""
                  src="/Images/feedback1.jpg"
                />
              </div>
              <div
                class="feedbacks__stars"
              >
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 12 12"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    clip-path="url(#clip0_1704_1143)"
                  >
                    <path
                      d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                      fill="#FDE82A"
                    />
                  </g>
                  <defs>
                    <clippath
                      id="clip0_1704_1143"
                    >
                      <rect
                        fill="white"
                        height="12"
                        width="12"
                      />
                    </clippath>
                  </defs>
                </svg>
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 12 12"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    clip-path="url(#clip0_1704_1143)"
                  >
                    <path
                      d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                      fill="#FDE82A"
                    />
                  </g>
                  <defs>
                    <clippath
                      id="clip0_1704_1143"
                    >
                      <rect
                        fill="white"
                        height="12"
                        width="12"
                      />
                    </clippath>
                  </defs>
                </svg>
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 12 12"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    clip-path="url(#clip0_1704_1143)"
                  >
                    <path
                      d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                      fill="#FDE82A"
                    />
                  </g>
                  <defs>
                    <clippath
                      id="clip0_1704_1143"
                    >
                      <rect
                        fill="white"
                        height="12"
                        width="12"
                      />
                    </clippath>
                  </defs>
                </svg>
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 12 12"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    clip-path="url(#clip0_1704_1143)"
                  >
                    <path
                      d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                      fill="#FDE82A"
                    />
                  </g>
                  <defs>
                    <clippath
                      id="clip0_1704_1143"
                    >
                      <rect
                        fill="white"
                        height="12"
                        width="12"
                      />
                    </clippath>
                  </defs>
                </svg>
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 12 12"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    clip-path="url(#clip0_1704_1143)"
                  >
                    <path
                      d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                      fill="#FDE82A"
                    />
                  </g>
                  <defs>
                    <clippath
                      id="clip0_1704_1143"
                    >
                      <rect
                        fill="white"
                        height="12"
                        width="12"
                      />
                    </clippath>
                  </defs>
                </svg>
              </div>
              <p
                class="feedbacks__card__name"
              >
                olena
              </p>
              <p
                class="feedbacks__card__description"
              >
                Small bags are my favorite in this season. They are perfect for parties and weekends when you don't have to wear many things.
              </p>
            </div>
            <div
              class="feedbacks__card"
            >
              <div
                class="feedbacks__image"
              >
                <img
                  alt=""
                  src="/Images/feedback2.jpg"
                />
              </div>
              <div
                class="feedbacks__stars"
              >
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 12 12"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    clip-path="url(#clip0_1704_1143)"
                  >
                    <path
                      d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                      fill="#FDE82A"
                    />
                  </g>
                  <defs>
                    <clippath
                      id="clip0_1704_1143"
                    >
                      <rect
                        fill="white"
                        height="12"
                        width="12"
                      />
                    </clippath>
                  </defs>
                </svg>
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 12 12"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    clip-path="url(#clip0_1704_1143)"
                  >
                    <path
                      d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                      fill="#FDE82A"
                    />
                  </g>
                  <defs>
                    <clippath
                      id="clip0_1704_1143"
                    >
                      <rect
                        fill="white"
                        height="12"
                        width="12"
                      />
                    </clippath>
                  </defs>
                </svg>
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 12 12"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    clip-path="url(#clip0_1704_1143)"
                  >
                    <path
                      d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                      fill="#FDE82A"
                    />
                  </g>
                  <defs>
                    <clippath
                      id="clip0_1704_1143"
                    >
                      <rect
                        fill="white"
                        height="12"
                        width="12"
                      />
                    </clippath>
                  </defs>
                </svg>
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 12 12"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    clip-path="url(#clip0_1704_1143)"
                  >
                    <path
                      d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                      fill="#FDE82A"
                    />
                  </g>
                  <defs>
                    <clippath
                      id="clip0_1704_1143"
                    >
                      <rect
                        fill="white"
                        height="12"
                        width="12"
                      />
                    </clippath>
                  </defs>
                </svg>
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 12 12"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    clip-path="url(#clip0_1704_1143)"
                  >
                    <path
                      d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                      fill="#FDE82A"
                    />
                  </g>
                  <defs>
                    <clippath
                      id="clip0_1704_1143"
                    >
                      <rect
                        fill="white"
                        height="12"
                        width="12"
                      />
                    </clippath>
                  </defs>
                </svg>
              </div>
              <p
                class="feedbacks__card__name"
              >
                semen
              </p>
              <p
                class="feedbacks__card__description"
              >
                Backpack is the best for me. These bags are suitable for many styles, from classics to modern minimalism.
              </p>
            </div>
            <div
              class="feedbacks__card"
            >
              <div
                class="feedbacks__image"
              >
                <img
                  alt=""
                  src="/Images/feedback3.jpg"
                />
              </div>
              <div
                class="feedbacks__stars"
              >
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 12 12"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    clip-path="url(#clip0_1704_1143)"
                  >
                    <path
                      d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                      fill="#FDE82A"
                    />
                  </g>
                  <defs>
                    <clippath
                      id="clip0_1704_1143"
                    >
                      <rect
                        fill="white"
                        height="12"
                        width="12"
                      />
                    </clippath>
                  </defs>
                </svg>
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 12 12"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    clip-path="url(#clip0_1704_1143)"
                  >
                    <path
                      d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                      fill="#FDE82A"
                    />
                  </g>
                  <defs>
                    <clippath
                      id="clip0_1704_1143"
                    >
                      <rect
                        fill="white"
                        height="12"
                        width="12"
                      />
                    </clippath>
                  </defs>
                </svg>
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 12 12"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    clip-path="url(#clip0_1704_1143)"
                  >
                    <path
                      d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                      fill="#FDE82A"
                    />
                  </g>
                  <defs>
                    <clippath
                      id="clip0_1704_1143"
                    >
                      <rect
                        fill="white"
                        height="12"
                        width="12"
                      />
                    </clippath>
                  </defs>
                </svg>
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 12 12"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    clip-path="url(#clip0_1704_1143)"
                  >
                    <path
                      d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                      fill="#FDE82A"
                    />
                  </g>
                  <defs>
                    <clippath
                      id="clip0_1704_1143"
                    >
                      <rect
                        fill="white"
                        height="12"
                        width="12"
                      />
                    </clippath>
                  </defs>
                </svg>
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 12 12"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    clip-path="url(#clip0_1704_1143)"
                  >
                    <path
                      d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                      fill="#FDE82A"
                    />
                  </g>
                  <defs>
                    <clippath
                      id="clip0_1704_1143"
                    >
                      <rect
                        fill="white"
                        height="12"
                        width="12"
                      />
                    </clippath>
                  </defs>
                </svg>
              </div>
              <p
                class="feedbacks__card__name"
              >
                anna
              </p>
              <p
                class="feedbacks__card__description"
              >
                Handbags with large fasteners and chains it’s for me. Such bags can be worn both on the shoulder and at the fingertips.
              </p>
            </div>
          </div>
          <div
            class="feedbacks__button"
          >
            <button
              class="button-white"
              type="button"
            >
              see more
            </button>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="feedbacks__wrapper"
    >
      <div
        class="container feedbacks"
      >
        <h2
          class="feedbacks__title"
        >
          feedbacks
        </h2>
        <div
          class="feedbacks__content"
        >
          <div
            class="feedbacks__card"
          >
            <div
              class="feedbacks__image"
            >
              <img
                alt=""
                src="/Images/feedback1.jpg"
              />
            </div>
            <div
              class="feedbacks__stars"
            >
              <svg
                fill="none"
                height="20"
                viewBox="0 0 12 12"
                width="20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  clip-path="url(#clip0_1704_1143)"
                >
                  <path
                    d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                    fill="#FDE82A"
                  />
                </g>
                <defs>
                  <clippath
                    id="clip0_1704_1143"
                  >
                    <rect
                      fill="white"
                      height="12"
                      width="12"
                    />
                  </clippath>
                </defs>
              </svg>
              <svg
                fill="none"
                height="20"
                viewBox="0 0 12 12"
                width="20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  clip-path="url(#clip0_1704_1143)"
                >
                  <path
                    d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                    fill="#FDE82A"
                  />
                </g>
                <defs>
                  <clippath
                    id="clip0_1704_1143"
                  >
                    <rect
                      fill="white"
                      height="12"
                      width="12"
                    />
                  </clippath>
                </defs>
              </svg>
              <svg
                fill="none"
                height="20"
                viewBox="0 0 12 12"
                width="20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  clip-path="url(#clip0_1704_1143)"
                >
                  <path
                    d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                    fill="#FDE82A"
                  />
                </g>
                <defs>
                  <clippath
                    id="clip0_1704_1143"
                  >
                    <rect
                      fill="white"
                      height="12"
                      width="12"
                    />
                  </clippath>
                </defs>
              </svg>
              <svg
                fill="none"
                height="20"
                viewBox="0 0 12 12"
                width="20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  clip-path="url(#clip0_1704_1143)"
                >
                  <path
                    d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                    fill="#FDE82A"
                  />
                </g>
                <defs>
                  <clippath
                    id="clip0_1704_1143"
                  >
                    <rect
                      fill="white"
                      height="12"
                      width="12"
                    />
                  </clippath>
                </defs>
              </svg>
              <svg
                fill="none"
                height="20"
                viewBox="0 0 12 12"
                width="20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  clip-path="url(#clip0_1704_1143)"
                >
                  <path
                    d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                    fill="#FDE82A"
                  />
                </g>
                <defs>
                  <clippath
                    id="clip0_1704_1143"
                  >
                    <rect
                      fill="white"
                      height="12"
                      width="12"
                    />
                  </clippath>
                </defs>
              </svg>
            </div>
            <p
              class="feedbacks__card__name"
            >
              olena
            </p>
            <p
              class="feedbacks__card__description"
            >
              Small bags are my favorite in this season. They are perfect for parties and weekends when you don't have to wear many things.
            </p>
          </div>
          <div
            class="feedbacks__card"
          >
            <div
              class="feedbacks__image"
            >
              <img
                alt=""
                src="/Images/feedback2.jpg"
              />
            </div>
            <div
              class="feedbacks__stars"
            >
              <svg
                fill="none"
                height="20"
                viewBox="0 0 12 12"
                width="20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  clip-path="url(#clip0_1704_1143)"
                >
                  <path
                    d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                    fill="#FDE82A"
                  />
                </g>
                <defs>
                  <clippath
                    id="clip0_1704_1143"
                  >
                    <rect
                      fill="white"
                      height="12"
                      width="12"
                    />
                  </clippath>
                </defs>
              </svg>
              <svg
                fill="none"
                height="20"
                viewBox="0 0 12 12"
                width="20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  clip-path="url(#clip0_1704_1143)"
                >
                  <path
                    d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                    fill="#FDE82A"
                  />
                </g>
                <defs>
                  <clippath
                    id="clip0_1704_1143"
                  >
                    <rect
                      fill="white"
                      height="12"
                      width="12"
                    />
                  </clippath>
                </defs>
              </svg>
              <svg
                fill="none"
                height="20"
                viewBox="0 0 12 12"
                width="20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  clip-path="url(#clip0_1704_1143)"
                >
                  <path
                    d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                    fill="#FDE82A"
                  />
                </g>
                <defs>
                  <clippath
                    id="clip0_1704_1143"
                  >
                    <rect
                      fill="white"
                      height="12"
                      width="12"
                    />
                  </clippath>
                </defs>
              </svg>
              <svg
                fill="none"
                height="20"
                viewBox="0 0 12 12"
                width="20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  clip-path="url(#clip0_1704_1143)"
                >
                  <path
                    d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                    fill="#FDE82A"
                  />
                </g>
                <defs>
                  <clippath
                    id="clip0_1704_1143"
                  >
                    <rect
                      fill="white"
                      height="12"
                      width="12"
                    />
                  </clippath>
                </defs>
              </svg>
              <svg
                fill="none"
                height="20"
                viewBox="0 0 12 12"
                width="20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  clip-path="url(#clip0_1704_1143)"
                >
                  <path
                    d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                    fill="#FDE82A"
                  />
                </g>
                <defs>
                  <clippath
                    id="clip0_1704_1143"
                  >
                    <rect
                      fill="white"
                      height="12"
                      width="12"
                    />
                  </clippath>
                </defs>
              </svg>
            </div>
            <p
              class="feedbacks__card__name"
            >
              semen
            </p>
            <p
              class="feedbacks__card__description"
            >
              Backpack is the best for me. These bags are suitable for many styles, from classics to modern minimalism.
            </p>
          </div>
          <div
            class="feedbacks__card"
          >
            <div
              class="feedbacks__image"
            >
              <img
                alt=""
                src="/Images/feedback3.jpg"
              />
            </div>
            <div
              class="feedbacks__stars"
            >
              <svg
                fill="none"
                height="20"
                viewBox="0 0 12 12"
                width="20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  clip-path="url(#clip0_1704_1143)"
                >
                  <path
                    d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                    fill="#FDE82A"
                  />
                </g>
                <defs>
                  <clippath
                    id="clip0_1704_1143"
                  >
                    <rect
                      fill="white"
                      height="12"
                      width="12"
                    />
                  </clippath>
                </defs>
              </svg>
              <svg
                fill="none"
                height="20"
                viewBox="0 0 12 12"
                width="20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  clip-path="url(#clip0_1704_1143)"
                >
                  <path
                    d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                    fill="#FDE82A"
                  />
                </g>
                <defs>
                  <clippath
                    id="clip0_1704_1143"
                  >
                    <rect
                      fill="white"
                      height="12"
                      width="12"
                    />
                  </clippath>
                </defs>
              </svg>
              <svg
                fill="none"
                height="20"
                viewBox="0 0 12 12"
                width="20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  clip-path="url(#clip0_1704_1143)"
                >
                  <path
                    d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                    fill="#FDE82A"
                  />
                </g>
                <defs>
                  <clippath
                    id="clip0_1704_1143"
                  >
                    <rect
                      fill="white"
                      height="12"
                      width="12"
                    />
                  </clippath>
                </defs>
              </svg>
              <svg
                fill="none"
                height="20"
                viewBox="0 0 12 12"
                width="20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  clip-path="url(#clip0_1704_1143)"
                >
                  <path
                    d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                    fill="#FDE82A"
                  />
                </g>
                <defs>
                  <clippath
                    id="clip0_1704_1143"
                  >
                    <rect
                      fill="white"
                      height="12"
                      width="12"
                    />
                  </clippath>
                </defs>
              </svg>
              <svg
                fill="none"
                height="20"
                viewBox="0 0 12 12"
                width="20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  clip-path="url(#clip0_1704_1143)"
                >
                  <path
                    d="M6.20634 0.5L7.55342 4.6459H11.9127L8.38597 7.2082L9.73305 11.3541L6.20634 8.7918L2.67963 11.3541L4.02671 7.2082L0.5 4.6459H4.85925L6.20634 0.5Z"
                    fill="#FDE82A"
                  />
                </g>
                <defs>
                  <clippath
                    id="clip0_1704_1143"
                  >
                    <rect
                      fill="white"
                      height="12"
                      width="12"
                    />
                  </clippath>
                </defs>
              </svg>
            </div>
            <p
              class="feedbacks__card__name"
            >
              anna
            </p>
            <p
              class="feedbacks__card__description"
            >
              Handbags with large fasteners and chains it’s for me. Such bags can be worn both on the shoulder and at the fingertips.
            </p>
          </div>
        </div>
        <div
          class="feedbacks__button"
        >
          <button
            class="button-white"
            type="button"
          >
            see more
          </button>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
